server/src/main/java/io/spine/server/integration/IntegrationBus.java

```diff
@@ -134,7 +134,7 @@
 
     @SuppressWarnings("ConstantConditions")     // `TransportFactory` has already been initialized.
     private IntegrationBus(Builder builder) {
-        final TransportFactory transportFactory = builder.getTransportFactory()
+        TransportFactory transportFactory = builder.getTransportFactory()
                                                          .get();
```

Comment:
Can you please review your changeset for the indentation in chained calls like this one? It's repeating over and over.